scripts/wrappers/join.py

```diff
@@ -594,10 +594,13 @@ def update_dqlite(cluster_cert, cluster_key, voters, host):
 
     # We get the dqlite port from the already existing deployment
     port = 19001
-    with open("{}/info.yaml".format(cluster_backup_dir)) as f:
-        data = yaml.safe_load(f)
-    if "Address" in data:
-        port = data["Address"].rsplit(":")[-1]
+    try:
+        with open("{}/info.yaml".format(cluster_backup_dir)) as f:
+            data = yaml.safe_load(f)
+        if "Address" in data:
+            port = data["Address"].rsplit(":")[-1]
+    except OSError:
+        pass
 
```

Comment:
this is unrelated to the changes of this PR, please revert

Reply:
Sure